apps/andi/lib/andi_web/validators/dataset_validator.ex

```diff
@@ -32,4 +37,21 @@ defmodule AndiWeb.DatasetValidator do
 
   defp same_system_name(a, b), do: a.technical.systemName == b.technical.systemName
   defp different_ids(a, b), do: a.id != b.id
+
+  defp description_required do
```

Comment:
Why do we have this function twice?

Reply:
That was us trying to merge in our change with @jakeprem's We have a bit of cleanup to do.